songs/admin.py

```diff
@@ -0,0 +1,25 @@
+from csv import list_dialects
+from django.contrib import admin
+from .models import (Artist, Track, Genre,
+                    Playlist, Album, PlaylistTrack)
+
+# Register your models here.
+
+admin.site.register(PlaylistTrack)
+
+@admin.register(Album)
+class AlbumAdmin(admin.ModelAdmin):
+    list_display = ('name', 'artist', 'created')
```

Comment:
Is 'artist' a foreign key on Album? If yes, then we need to remove it from list_display because when the app grows, foreign key listings can become very slow. So we avoid adding them in admin panel

Reply:
removed all the foreign key listings in admin panel.